authentication-ldap/build.gradle

```diff
@@ -1,13 +1,22 @@
 ext.moduleName = 'com.synopsys.integration.alert.authentication.ldap'
 
+repositories {
+    // Need this repository to resolve the spring-security-saml2-core dependencies
+    maven {
+        url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
```

Comment:
Why is this repository needed in this project?  The component project dependency?   Is it the use of the AuthenticationDescriptorKey?

Reply:
Should not have been included. Has been removed in latest commit.